src/RelayClient.ts

```diff
@@ -271,7 +273,7 @@ class RelayClient extends EnvelopingEventEmitter {
   ): boolean {
     return (
       to != constants.AddressZero &&
-      data === '0x00' &&
+      isDataEmpty(data.toString()) &&
       BigNumber.from(value).isZero()
```

Comment:
According to [this comment](https://github.com/rsksmart/rif-relay-client/pull/91#discussion_r1667727589), we may probable remove the line `BigNumber.from(value).isZero()`.